content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/expiration_dates.rst

```diff
@@ -1,65 +1,230 @@
+
+Set expiration dates on manufactured products
+=============================================
+
+Expiration dates can also be generated for products manufactured in-house. To assign expiration
+dates to manufactured products, a manufacturing order (MO) needs to be completed.
+
+To create a :abbr:`MO (Manufacturing Order)`, go to :menuselection:`Manufacturing --> Operations
+--> Manufacturing Orders`, and click :guilabel:`Create`. Choose a product to manufacture from the
+:guilabel:`Product` :guilabel:`drop-down` menu, and choose a :guilabel:`Quantity` to produce.
+
+.. image:: expiration_dates/expiration-dates-manufacturing-order.png
    :align: center
+   :alt: Manufacturing order for product with expiration date.
+
+.. note::
+  To manufacture a product, there need to be materials :guilabel:`To Consume` in the
+  :guilabel:`Product` lines. This can be achieved either by creating a :guilabel:`Bill of Material`
+  for the :guilabel:`Product`, or manually adding materials to consume by clicking
+  :guilabel:`Add a line`.
 
-.. image:: expiration_dates/expiration_dates_04.png
+Click :guilabel:`Confirm`. Then, next to :guilabel:`Lot/Serial Number`, either select an existing
+lot number from the :guilabel:`drop-down` menu, or click the green :guilabel:`+` sign to
+automatically assign a new lot number. Then, mark the done :guilabel:`Quantity`, and
+:guilabel:`Mark as Done`.
+
+Click on the assigned :guilabel:`Lot/Serial Number` to be taken to a form for that number. Under
+the :guilabel:`Dates` tab, all expiration information previously configured for the product will be
+available. The same information is also available directly from the product form, or by going to
+:menuselection:`Inventory --> Products --> Lots/Serial Numbers`.
+
+.. image:: expiration_dates/expiration-dates-dates-tab-lot-number.png
+   :align: center
+   :alt: Dates tab with expiration information for specific lot number.
+
+Sell products with expiration dates
+===================================
+
+.. note::
+  To sell perishable products with expiration dates, the :guilabel:`Removal Strategy` for the
+  :guilabel:`Location` the products are stored in must be set to
+  :abbr:`FEFO (First Expiry, First Out)`. If there is not enough stock of perishable products in
+  one lot, Odoo will automatically take the remaining quantity required from a second lot with the
+  next-soonest expiration date. Removal strategies can also be set on
+  :guilabel:`Product Categories`.
+
+.. seealso::
+    - :doc:`../../routes/strategies/removal`
+
+Selling perishable products with expiration dates is done the same as any other type of product.
+The first step in selling perishable products is to create a sales order. To do that, go to
+:menuselection:`Sales --> Create` to create a new quotation, and fill out the information. Add a
+:guilabel:`Customer`, click :guilabel:`Add a product` to add products to the :guilabel:`Product`
+lines, then set a :guilabel:`Quantity`.
```

Comment:
Selling perishable products with expiration dates is done the same as any other type of product.
The first step in selling perishable products is to create a sales order. 

To do that, go to :menuselection:`Sales application --> Create` to create a new quotation, and fill out the information on the sales order form. 

Add a :guilabel:`Customer`, click :guilabel:`Add a product` to add the desired products to the :guilabel:`Product`
lines, then set a :guilabel:`Quantity`.